trpcultivate_phenotypes/src/Service/TripalCultivatePhenotypesExperimentService.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * @file
+ * Tripal Cultivate Phenotypes Experiments/Projects service definition.
+ */
+
+namespace Drupal\trpcultivate_phenotypes\Service;
+
+/**
+ * Class TripalCultivatePhenotypesExperimentService.
+ */
+class TripalCultivatePhenotypesExperimentService {
+  
+  /**
+   * Chado Db.
+   */
+  protected $chado;
+
+  /**
+   * Constructor.
+   */
+  public function __construct() {
+    // Chado database.
+    $this->chado = \Drupal::service('tripal_chado.database');
```

Comment:
This should use dependency injection to get the database if you determine this service is still needed.